app/root.tsx

```diff
@@ -1,19 +1,40 @@
-import type { LinksFunction, LoaderArgs } from '@remix-run/node';
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+  isRouteErrorResponse,
+  useRouteError,
+} from '@remix-run/react';
+import type { LinksFunction, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
 import { json } from '@remix-run/node';
 import stylesheet from './styles/tailwind.css';
 import { TopNav } from '~/components/layout/top-nav';
-import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
 import Footer from './components/layout/footer';
+import { ErrorMessage } from '~/components/ui/error-message';
 import { getCurrentUser } from '~/modules/session/session.server';
 import { db } from './modules/database/db.server';
 
+export const meta: MetaFunction = ({ error }: any) => {
+  const title = !error
+    ? 'Plan It Social'
+    : isRouteErrorResponse(error)
```

Comment:
Should we use the `error.status` property to give the user some more concrete information/context about **what** error occurred?

For instance: 404 -> "Page not found"!

Reply:
(No blocker, can be follow-up work!)